Build problem with Vite

**Describe the bug**
In a Typescript Vue 3 Vite project and I get an error importing the package. I see the following error:

 [ERROR] Cannot assign to import "download"

    node_modules/vue-papa-parse/src/index.js:46:9:
      46 │     Papa.download = _downloadCsv
         ╵          ~~~~~~~~

  Imports are immutable in JavaScript. To modify the value of this import, you must export a setter
  function in the imported file (e.g. "setDownload") and then import and call that function here
  instead.

✘ [ERROR] Cannot assign to import "dedupe"

    node_modules/vue-papa-parse/src/index.js:47:9:
      47 │     Papa.dedupe = _dedupe
         ╵          ~~~~~~

  Imports are immutable in JavaScript. To modify the value of this import, you must export a setter
  function in the imported file (e.g. "setDedupe") and then import and call that function here
  instead.

Does the package support Vite?

Awesome thanks @twickstrom really appreciate it. 

I also see the same error on with download. 

 [ERROR] Cannot assign to import "download"

    node_modules/vue-papa-parse/src/index.js:46:9:
      46 │     Papa.download = _downloadCsv
         ╵          ~~~~~~~~

  Imports are immutable in JavaScript. To modify the value of this import, you must export a setter
  function in the imported file (e.g. "setDownload") and then import and call that function here
  instead.

Hi @twickstrom, I've run into this issue too. Replication steps:

1. Create a new Vue 3 + Vite project
```bash
yarn create vite
```
```bash
✔ Project name: … vite-project
✔ Select a framework: › vue
✔ Select a variant: › vue-ts

Scaffolding project in vite-project...

Done. Now run:

  cd vite-project
  yarn
  yarn dev

✨  Done in 15.63s.
```

2. Add `vue-papa-parse` as a dependency and register it as a plugin
```bash
yarn add vue-papa-parse
```
`main.ts`
```typescript
import { createApp } from 'vue'
import App from './App.vue'

import VuePapaParse from 'vue-papa-parse'

const app = createApp(App)

app.use(VuePapaParse)
app.mount('#app')

```

3. Run the Vite dev server, receive an error
```bash
yarn dev
```
output:
```bash
yarn run v1.22.15
$ vite

  vite v2.8.4 dev server running at:

  > Local: http://localhost:XXXX/
  > Network: use `--host` to expose

  ready in 315ms.

XX:XX:XX [vite] new dependencies found: vue-papa-parse, updating...
✘ [ERROR] Cannot assign to import "download"

    node_modules/vue-papa-parse/src/index.js:46:9:
      46 │     Papa.download = _downloadCsv
         ╵          ~~~~~~~~

  Imports are immutable in JavaScript. To modify the value of this import, you must export a setter
  function in the imported file (e.g. "setDownload") and then import and call that function here
  instead.

✘ [ERROR] Cannot assign to import "dedupe"

    node_modules/vue-papa-parse/src/index.js:47:9:
      47 │     Papa.dedupe = _dedupe
         ╵          ~~~~~~

  Imports are immutable in JavaScript. To modify the value of this import, you must export a setter
  function in the imported file (e.g. "setDedupe") and then import and call that function here
  instead.

XX:XX:XX [vite] error while updating dependencies:
Error: Build failed with 2 errors:
node_modules/vue-papa-parse/src/index.js:46:9: ERROR: Cannot assign to import "download"
node_modules/vue-papa-parse/src/index.js:47:9: ERROR: Cannot assign to import "dedupe"
```

--- a/package.json
+++ b/package.json
@@ -42,6 +42,7 @@
   "homepage": "https://github.com/twickstrom/vue-papa-parse",
   "main": "src/index.js",
   "dependencies": {
+    "lodash": "^4.17.21",
     "papaparse": "^5.3.0"
   },
   "peerDependencies": {

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vue-papa-parse",
   "description": "A simple wrapper for for the original PapaParse built for VueJs. Supports both Vue 2 & Vue 3.",
-  "version": "3.0.4",
+  "version": "3.0.5",
   "readme": "README.md",
   "author": {
     "name": "Tim Wickstrom",

--- a/src/index.js
+++ b/src/index.js
@@ -43,12 +43,13 @@ const _dedupe = (arr) => {
 
 const VuePapaParse = {
   install (app, options) {
-    Papa.download = _downloadCsv
-    Papa.dedupe = _dedupe
+    let localPapa = Papa
+    localPapa.download = _downloadCsv
+    localPapa.dedupe = _dedupe
     if ('config' in app && 'globalProperties' in app.config) {
-      app.config.globalProperties.$papa = Papa
+      app.config.globalProperties.$papa = localPapa
     } else {
-      app.prototype.$papa = Papa
+      app.prototype.$papa = localPapa
     }
   }
 }

--- a/src/index.js
+++ b/src/index.js
@@ -1,5 +1,5 @@
 import * as Papa from 'papaparse'
-
+import { cloneDeep } from 'lodash'
 const _downloadCsv = (csv, title) => {
   try {
     let csvData = new Blob([csv], { type: 'text/csv' })
@@ -43,7 +43,7 @@ const _dedupe = (arr) => {
 
 const VuePapaParse = {
   install (app, options) {
-    let localPapa = Papa
+    let localPapa = cloneDeep(Papa)
     localPapa.download = _downloadCsv
     localPapa.dedupe = _dedupe
     if ('config' in app && 'globalProperties' in app.config) {

